02LessonTwo/homework/index.html

```diff
@@ -61,19 +61,19 @@
     <center><h2>Hobbies</h2></center>
       <center>
         <div id="hobbies">
-          <img src="http://www.engelbert-schmid-horns.com/images/hoerner/01-Naturhorn2.jpg" height="20%" width="20%">
-          <img src="https://www.insurancehotline.com/wp-content/uploads/2014/01/dreamstime_xl_31762033.jpg" height="21%" width="21%">
-          <img src="http://d1l9zs272jkxri.cloudfront.net/blog/uploads/2015/01/paris-cover-2.jpg" height="26%" width="26%">
-          <img src="http://media.cutimes.com/cutimes/article/2016/07/03/technology1.jpg" height="22%" width="22%">
+          <img class="hobbies" src="http://www.engelbert-schmid-horns.com/images/hoerner/01-Naturhorn2.jpg" height="20%" width="20%">
+          <img class="hobbies" src="https://www.insurancehotline.com/wp-content/uploads/2014/01/dreamstime_xl_31762033.jpg" height="21%" width="21%">
+          <img class="hobbies" src="http://d1l9zs272jkxri.cloudfront.net/blog/uploads/2015/01/paris-cover-2.jpg" height="26%" width="26%">
+          <img class="hobbies" src="http://media.cutimes.com/cutimes/article/2016/07/03/technology1.jpg" height="22%" width="22%">
         </div>
```

Comment:
Pretty impressed with the overall work you did. Definitely one of the best Bio pages I have seen in class. Have you done this before? :smiley: